### DISCUSSION TITLE
Can't load/download extensions

### DISCUSSION THREAD

**Discussion by Unrepentant-Atheist** [coder/code-server#5629] [Q&A] [answered]
I'm using https://hub.docker.com/r/linuxserver/code-server on my local server, only in my LAN. My server, where the container runs, does have internet access.

Whenever I try to download extensions I get

![image](https://user-images.githubusercontent.com/3938423/194706434-515b9339-524d-484c-ad53-b29458068f1a.png)

What's going on?

**Comment by lg-nineoneninethreenine** [comment]
Did you check which server it connects to? We got a new marketplace, so it could be a connect error, right?

**Comment by jsjoeio** [comment, marked as answer]
I think Open VSX was down temporarily. Should be up now.

**Reply by Unrepentant-Atheist** [reply]
Maybe, dunno, solution for me was to switch to image: gitpod/openvscode-server and be done with it.

But if you say it was Open VSX , then maybe I ought to fire it up again and change the config to:

      "extensionsGallery": {
        "nlsBaseUrl": "https://www.vscode-unpkg.net/_lp/",
        "serviceUrl": "https://marketplace.visualstudio.com/_apis/public/gallery",
        "cacheUrl": "https://vscode.blob.core.windows.net/gallery/index",
        "itemUrl": "https://marketplace.visualstudio.com/items",
        "publisherUrl": "https://marketplace.visualstudio.com/publishers",
        "resourceUrlTemplate": "https://{publisher}.vscode-unpkg.net/{publisher}/{name}/{version}/{path}",
        "controlUrl": "https://az764295.vo.msecnd.net/extensions/marketplace.json",
        "recommendationsUrl": "https://az764295.vo.msecnd.net/extensions/workspaceRecommendations.json.gz"
      },

I always do this when I update https://github.com/VSCodium/vscodium , Open VSX is a shite marketplace compared to established vscode market. (still, unfortunately)
